Storage/src/StreamWrapper.php

```diff
@@ -127,6 +127,24 @@ public function stream_set_option()
         return false;
     }
 
+    /**
+     * This is called when touch is used on a stream. See:
+     * https://www.php.net/manual/en/streamwrapper.stream-metadata.php
+     */
+    public function stream_metadata($path, $option, $value)
+    {
+        if ($option == STREAM_META_TOUCH) {
```

Comment:
So, you are only considering the `touch` as an operation? How about `chmod`, `chown`, `chgrp`? Are they not supported by storage?

Reply:
touch not working => this was actually reported by a customer. So, for now chmod, chown and chgrp are out of scope.

Reply:
Spotted one issue, where by current implementation would've overwritten an existing file on touch. Fixed it in 8fec964fbb2b41571b27a7b5843bcf1477a7bf60